common/proto/HStream/Server/HStreamApi.proto

```diff
@@ -297,9 +297,18 @@ message GetQueryRequest {
   string id = 1;
 }
 
+enum Status {
```

Comment:
How about `QueryStatus`?

Reply:
The status is used by connectors, queries, and views.